pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.2
+2021-09-02 (Date of Last Commit)
+
+* Changed the StarSolo task for Optimus to align from the FASTQ file
```

Comment:
Assuming these changes are the same changes that are listed in the Multisample changelog, you can use the same changelog entry

Reply:
It was just StarAlign.wdl. We have sctools for metrics and we are not changing Picard.